### PR TITLE
fix(form): support legacy input names on new form components

### PR DESCRIPTION
**Pull request by pweyrich**
@plentymarkets/team-terra

#### Must be done by Terra

Testing

-   [ ] Person 1 (mandatory)

    Browser support (relevant for changes in scss / appearance of components)

    -   [ ] Chrome
    -   [ ] Firefox
    -   [ ] Safari

    Responsiveness

    -   [ ] Desktop
    -   [ ] Tablet
    -   [ ] Mobile

-   [ ] Person 2 (mandatory)

    Browser support

    -   [ ] Chrome
    -   [ ] Firefox
    -   [ ] Safari

    Responsiveness

    -   [ ] Desktop
    -   [ ] Tablet
    -   [ ] Mobile

Information transfer

-   [ ] Inform Ceres about changes in `TerraFormComponent` (optional: relevant for changes in this component)

Terra Basic Plugin

-   [ ] Adapt changes (optional: relevant for version updates, global design changes, l10n)

---

#### Must be done by every developer

Please inform a member of Terra (@plentymarkets/team-terra) to get the upper part of the checklist done (with urgency or deadline).

Documentation

-   [ ] Changelog (optional: relevant for every change which influences the API)
-   [ ] Example (updated or created)
-   [ ] JSDoc (optional: relevant for every change which influences the API)
-   [ ] [Google spreadsheet](https://docs.google.com/spreadsheets/d/1OINnux8TEoitV-qdAxqUQaf9oGI_fqFVwfRWenRFfhI/edit#gid=0) (relevant when deprecating public APIs or features)

Testing

-   [ ] Unit Test (updated or created)

### CODE REVIEW

File: src/lib/components/forms/form/form-entry/terra-form-entry.base.ts [hunk 1]
```diff
@@ -142,4 +147,15 @@ export class TerraFormEntryBase implements OnChanges, OnDestroy {
...
+
+    /**
+     * Removes the prefix 'input' from the given propertyName and lowercases the succeeding character.
+     * @param propertyName
+     */
+    private _removeInputPrefix(propertyName: string): string {
+        if (!propertyName.startsWith('input')) {
+            return propertyName;
+        }
+        return propertyName.charAt(5).toLowerCase() + propertyName.substr(6);
```
Comment (dantrau) [inline src/lib/components/forms/form/form-entry/terra-form-entry.base.ts:159]: optional aber wegen der magic number
```suggestion
        return propertyName.charAt('input'.length).toLowerCase() + propertyName.substr(6);
```

**Review by dantrau — APPROVED** [review]

**Comment by dantrau** [comment]
not tested

File: src/lib/components/forms/form/form-entry/terra-form-entry.base.ts (hunk 1 above)
Comment (pweyrich) [reply src/lib/components/forms/form/form-entry/terra-form-entry.base.ts:159]: ich habe mich bewusst dagegen entschieden. Ich hätte es wenn nämlich ganzheitlich (mit separater Konstante, etc..) gelöst und dafür auch andere Stellen anpassen müssen. Das wollte ich vermeiden.

**Review by zhajjhassan — APPROVED** [review]
tested

**Pull request merged**